### ISSUE TITLE
add windows suport

### ISSUE THREAD

**Issue by insinfo** [vknabel/vscode-swiftformat#30]
I have swift 5.9 and swiftformat 0.52.7 installed but when I use the format option in VS Code it deletes all source code

![image](https://github.com/vknabel/vscode-swiftformat/assets/12227024/35bf7ebc-9a76-4788-8332-7a2d09fcc21c)

![image](https://github.com/vknabel/vscode-swiftformat/assets/12227024/d35bc0fe-2aa1-4d65-a745-26a14d9c3bd2)

**Comment by vknabel** [comment]
Thanks for the report! As I don't have access to a windows system, I am unable to reproduce your problem.

As debugging windows issues requires lots of ping pong with a tester, I would appreciate a pull request by any volunteer.

**Comment by MChriss12** [comment]
Still a bug. It deletes all the source code when trying to autoformat on save.

**Comment by vknabel** [comment]
Correct. I haven’t done anything, because as mentioned, I don’t have access to a device running windows.

I would appreciate a pull request by any volunteer.

Alternatively I could simply disable all formatting on windows.

**Comment by oskargargas** [comment]
Take a look at https://github.com/vknabel/vscode-apple-swift-format/issues/40#issuecomment-1870430614
same fix works for `swiftformat` too
